docs/documentation_landing.md

```diff
@@ -0,0 +1,36 @@
+## Charmed MySQL Documentation
+The Charmed MySQL Operator delivers automated operations management from [day 0 to day 2](https://codilime.com/blog/day-0-day-1-day-2-the-software-lifecycle-in-the-cloud-age/) on the [MySQL Community Edition](https://www.mysql.com/products/community/) relational database. It is an open source, end-to-end, production-ready data platform [on top of Juju](https://juju.is/).
+
+MySQL is the world’s most popular open source database. A relational database stores data in separate tables rather than putting all the data in one big storeroom. The database structure is organized into physical files optimized for speed. The logical data model, with objects such as data tables, views, rows, and columns, offers a flexible programming environment.
+
+This MySQL operator charm comes in two flavours to deploy and operate MySQL on [physical/virtual machines](https://github.com/canonical/mysql-operator) and [Kubernetes](https://github.com/canonical/mysql-k8s-operator). Both offer features such as replication, TLS, password rotation, and easy to use integration with applications. The Charmed MySQL Operator meets the need of deploying MySQL in a structured and consistent manner while allowing the user flexibility in configuration. It simplifies deployment, scaling, configuration and management of MySQL in production at scale in a reliable way.
+
+[note type="positive"]
+**"Charmed MySQL" or "Mysql Innodb Cluster"?**
```

Comment:
iirc the official stance is all things "charmed"

Reply:
The idea here is to describe list already available MySQL/MariaDB charms explaining "Charmed MySQL" replaces all of them.

Reply:
Please check the new list of charms mentioned there, I have tried to make it better readable.